sdks/python/apache_beam/transforms/external.py

```diff
@@ -433,6 +434,9 @@ def __init__(self, urn, payload, expansion_service=None):
     self._inputs = {}  # type: Dict[str, pvalue.PCollection]
     self._outputs = {}  # type: Dict[str, pvalue.PCollection]
 
+  def with_output_types(self, *args, **kwargs):
```

Comment:
Is this needed? (Shouldn't it be inherited?)

Reply:
Without it, the default implementation only allows a single argument.